.github/workflows/dvrescue-gui.yml

```diff
@@ -0,0 +1,327 @@
+name: dvrescue-gui
```

Comment:
I am not in favor of so many builds, IMO it is fine enough with default compiler on each platform (so 3 jobs in total).

Reply:
okay, I'll reduce it. But I also planned to introduce linux pkg-config build (like MacOS brew one). So, in total it will be

1. MacOS - brew
2. MacOS
3. Linux - pkgconfig
4. Linux
5. Windows - msvc
6. Windows - mingw (?)

Without introducing 'linux - pkgconfig' and without 'windows - mingw' it might be just 4 configurations. Also, if caching will properly be enabled - build times should be reduced significantly.